app/views/users/feed.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>
+  <%= @user.username %>
+</h1>
+
+<div class="row">
```

Comment:
Everything looks good! I ended up making a partial for the user navbar to save some space and so I'd only have to write out the links once, if you have some spare time I'd suggest doing that!

Reply:
Thanks, I saw that on yours; super smart!